simpleio.py

```diff
@@ -101,6 +102,66 @@ def shift_out(dataPin, clock, value, msb_first=True):
             tmpval = bool((value & (1 << i)))
             dataPin.value = tmpval
 
+class Servo:
+    """
+    Easy control for hobby (3-wire) servos
+
+    :param ~pulseio.PWMOut pin: PWM pin where the servo is located.
```

Comment:
This should be `microcontroller.Pin` I believe. (Thats what items in `board` are.)